packages/subgraph/src/dao/dao_v1_0_0.ts

```diff
@@ -164,49 +163,52 @@ export function handleNativeTokenDeposited(event: NativeTokenDeposited): void {
 }
 
 export function handleGranted(event: Granted): void {
-  // ContractPermissionId
-  let daoId = event.address.toHexString();
-  let contractPermissionIdEntityId =
-    event.params.where.toHexString() +
-    '_' +
-    event.params.permissionId.toHexString();
-  let contractPermissionIdEntity = ContractPermissionId.load(
-    contractPermissionIdEntityId
-  );
-  if (!contractPermissionIdEntity) {
-    contractPermissionIdEntity = new ContractPermissionId(
-      contractPermissionIdEntityId
-    );
-    contractPermissionIdEntity.dao = daoId;
-    contractPermissionIdEntity.where = event.params.where;
-    contractPermissionIdEntity.permissionId = event.params.permissionId;
-    contractPermissionIdEntity.save();
-  }
+  const contractAddress = event.address.toHexString();
+  const where = event.params.where;
+  const contractPermissionId = event.params.permissionId;
```

Comment:
Why `contractPermissionId` and not just `permissionId`?

Reply:
perhaps some remnant of the past,
amended